users/charlesrocket/rules.mk

```
@@ -0,0 +1,3 @@
+ifdef APL_ENABLE
```

You might want this instead: 
```suggestion
ifeq ($(strip $(APL_ENABLE)), yes)
```

---

Thanks!